### DISCUSSION TITLE
Points and Tags

### DISCUSSION THREAD

**Discussion by Blackscreen21** [Traewelling/traewelling#2739] [Ideas / Feature Requests]
### Describe the bug

The screen which displays the route and the points for it is not showing after check-in. Meaning the user cannot see how many points they received.
Also the tags do not show in either the dashboard nor the profile, so they are pretty useless as of now.

### Steps to reproduce

Check-in with tags.
No overview screen.
Tags dissapear.

### Browser

Chrome

**Comment by pascalwittler** [comment]
Tags currently only appear after clicking on the status itself – possible overlap with #2655

**Comment by pascalwittler** [comment]
Although, I can confirm the first part (no more points, no more kilometers, and no more other people who checked in in the green success message box) since the new check-in dialog was released.

**Comment by MrKrisKrisu** [comment]
> so they are pretty useless as of now.

I would like to deny that they are useless. They are simply not displayed excessively in the UI, but are present in the export and are a very powerful feature for statistics and co.
